src/libs/PolicyUtils.ts

```diff
@@ -528,6 +528,17 @@ function getNetSuiteTaxAccountOptions(policy: Policy | undefined, subsidiaryCoun
         }));
 }
 
+function isSyncTaxEnabled(policy: Policy | undefined): boolean {
```

Comment:
This is trying to do the same thing as `canUseTaxNetSuite` below - can we just reuse that method instead? Also the name is misleading here.

Reply:
I've updated to use `canUseNetSuiteUSATax `